### DISCUSSION TITLE
Netmiko enters to conf t  on Cisco ASA  for show command

### DISCUSSION THREAD

**Discussion by rltek2000** [ktbyers/netmiko#3124] [General]
New to netmiko. In short, noticed that my script which was performed 'show license' command in exec mode (not conf t) actually causing config change on ASAv -- and debugging show its enters to conf t for some reason.
While interested  command output was just one bellow. Is there a way to avoid config change and entering into 'conf t'?
code:
         CiscoASA = {
            'device_type': 'cisco_asa',
            'ip': ip,
            'username': username,
            'password': password,
            'port': 22,
            'auth_timeout': 10,
            'timeout': 10,
            'verbose': False,
            }
         fail=False
         try:
             net_connect = ConnectHandler(**CiscoASA)
            #net_connect.ansi_escape_codes = True
         except (AuthenticationException):
             print (hostname,",Authentication failure: " + ip)
             fail=True
             pass
       
         if not fail:
            output = net_connect.send_command("show license status",read_timeout=10)

**Comment by ktbyers** [comment]
You can pass the argument `auto_allow_change = False`.

Note, in general long commands and Netmiko will break (if the terminal width is not increased). Long commands being commands where the prompt + command is longer than about 80 characters.
